src/auth.nim

```diff
@@ -24,6 +25,11 @@ const
   }.toTable
 
 var
+  pool: HttpPool
+  guestAccountsUrl = ""
+  guestAccountsHost = ""
+  guestAccountsKey = ""
```

Comment:
You'll can access these from the cfg object. I suggest setting up a `while true` async loop that tries to get accounts maybe once an hour (tweakable), initialized from the main nitter.nim file. It would be similar to how the token pool used to work: 

https://github.com/zedeus/nitter/blob/b62d73dbd373f08af07c7a79efcd790d3bc1a49c/src/nitter.nim#L41

https://github.com/zedeus/nitter/blob/b62d73dbd373f08af07c7a79efcd790d3bc1a49c/src/tokens.nim#L145-L166

Reply:
done